### DISCUSSION TITLE
Feature Request Follow Organization

### DISCUSSION THREAD

**Discussion by ghost** [community/community#3053] [Other Feature Feedback, Questions, & Ideas]
Can you please add a GitHub feature to follow "organization" not just specific "repository"?
For example, I would like to be able to follow; https://github.com/github  similarly like I do with "repositories". Thank you!

**Comment by jfullstackdev** [comment]
it's already available, there is the follow button 

![image](https://user-images.githubusercontent.com/47092464/182751267-63f0ba2a-c654-46ee-94bd-48c576e33810.png)
